usaspending_api/references/v2/views/recipients.py

```diff
@@ -151,6 +154,7 @@ def _parse_elasticsearch_response(self, hits: Dict[str, Any]) -> List[Dict[str,
                         ("recipient_name", recipient["recipient_name"]),
                         ("uei", recipient["uei"]),
                         ("recipient_level", recipient["recipient_level"]),
+                        ("duns", recipient["duns"] if "duns" in recipient else None),
```

Comment:
Is `recipient` a dict? If so you can probably use the `.get()` syntax to spare the `if else` check. If `recipient` is something special with ES though you can disregard.

Reply:
there is no .get() function but I can access the variable directly such as `recipient.duns` which i guess works the same

Reply:
Nope, i was mistaken, it needs to be `recipient["duns"] if "duns" in recipient else None`